first.js

```diff
@@ -0,0 +1,118 @@
+//const getButton = document.querySelector("button");
+let allTasks = [];
+let valueInput = "";
+//let input = null;
+let activatedEditTask = null;
+
+window.onload = async () => {
+  input = document.getElementById("input-todo");
+  if (input === null) {
+    return }
+    input.addEventListener("change", updateValue);
+    // localStorage.setItem("tasks", JSON.stringify(allTasks));
+    const resp = await fetch("http://localhost:3000/allTasks", {
```

Comment:
http://localhost:3000 is repeated many times, can be moved to a separate variable